TopKRdbLoad L275 overwrites the heap allocated string causing a small leak on that scenario

Using the valgrind CI PR that I'll open we can detect the following issue:
```
	test_topk:testTopK.test_validation
1618:M 20 Feb 2021 23:40:10.984 * DB saved on disk
	[PASS]
1618:M 20 Feb 2021 23:40:11.018 * DB saved on disk
1618:signal-handler (1613864411) Received SIGTERM scheduling shutdown...
1618:M 20 Feb 2021 23:40:11.101 # User requested shutdown...
1618:M 20 Feb 2021 23:40:11.105 * Saving the final RDB snapshot before exiting.
1618:M 20 Feb 2021 23:40:11.113 * DB saved on disk
1618:M 20 Feb 2021 23:40:11.120 # Redis is now ready to exit, bye bye...
==1618== 20 bytes in 20 blocks are definitely lost in loss record 282 of 1,111
==1618==    at 0x4C31B0F: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==1618==    by 0x156835: ztrymalloc_usable (zmalloc.c:92)
==1618==    by 0x1568D6: ztrymalloc (zmalloc.c:117)
==1618==    by 0x17DB5A: rdbGenericLoadStringObject (rdb.c:533)
==1618==    by 0x1F25B9: moduleLoadString (module.c:4587)
==1618==    by 0x1F262A: RM_LoadStringBuffer (module.c:4618)
==1618==    by 0x69D1F8E: TopKRdbLoad (rm_topk.c:273)
==1618==    by 0x182F7D: rdbLoadObject (rdb.c:2198)
==1618==    by 0x183FE1: rdbLoadRio (rdb.c:2534)
==1618==    by 0x184427: rdbLoad (rdb.c:2643)
==1618==    by 0x1AC9A1: debugCommand (debug.c:537)
==1618==    by 0x14CF0B: call (server.c:3675)
==1618== 
	bad exit code for serverId 1
	Redis did not exit cleanly
Test Took: 10 sec
Total Tests Run: 8, Total Tests Failed: 1, Total Tests Passed: 7
Failed Tests Summary:
	test_topk.__init__
		redis process failure
Makefile:79: recipe for target 'test' failed
make: *** [test] Error 1
```
Link to the automatic CI check: https://app.circleci.com/pipelines/github/RedisBloom/RedisBloom/347/workflows/40b7ea20-9b06-4588-9e87-a0c89f25f4f6/jobs/1699
The PR that adds valgrind to CI already includes the fix for this small issue.

--- a/src/rm_topk.c
+++ b/src/rm_topk.c
@@ -272,6 +272,7 @@ static void *TopKRdbLoad(RedisModuleIO *io, int encver) {
     for (uint32_t i = 0; i < topk->k; ++i) {
         topk->heap[i].item = RedisModule_LoadStringBuffer(io, &itemSize);
         if (itemSize == 1) {
+            RedisModule_Free(topk->heap[i].item);
             topk->heap[i].item = NULL;
         }
     }

